docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-overview.md

```diff
@@ -94,23 +94,47 @@ Cloud providers offer a variety of instance types across the regions where they
 
 ### Fault tolerance
 
-The _fault tolerance_ determines how resilient the cluster is to node, zone, and region failures. YugabyteDB Managed provides the following options for providing replication and redundancy:
+YugabyteDB achieves resilience by replicating data across fault domains using the [RAFT consensus protocol](../../../architecture/docdb-replication/replication/). The fault domain can be at the level of individual nodes, availability zones, or entire regions.
 
-- **Region Level**. Includes 3 nodes spread across multiple regions with a [replication factor](../../../architecture/docdb-replication/replication/) (RF) of 3. YugabyteDB can continue to do reads and writes even in case of a cloud region failure. This configuration provides the maximum protection for a regional failure.
+The _fault tolerance_ determines how resilient the cluster is to domain (that is, node, zone, or region) outages. Fault tolerance is achieved by adding redundancy, in the form of additional nodes, across the fault domain. Due to the way the RAFT protocol works, providing a fault tolerance of `ft` requires replicating data across `2ft + 1` domains. For example, to survive the outage of 2 nodes, a cluster needs 2 * 2 + 1 nodes. While the 2 nodes are offline, the remaining 3 nodes can continue to serve reads and writes without interruption.
 
-- **Availability Zone Level**. Includes a minimum of 3 nodes spread across multiple availability zones with a RF of 3. YugabyteDB can continue to do reads and writes even in case of a cloud availability zone failure. This configuration provides the maximum protection for a data center failure.
+YugabyteDB Managed provides the following configurations for fault tolerance.
 
-- **Node Level**. Includes a minimum of 3 nodes deployed in a single availability zone with a RF of 3. YugabyteDB can continue to do reads and writes even in case of a node failure, but this configuration is not resilient to cloud availability zone or region outages.
+| Fault tolerance | Resilient to | Minimum number of nodes | Scale in increments of |
```

Comment:
"fault tolerance" should be renamed "AZ or regional fault tolerance"  if we use "none" as the level . 

Alternatively, "none" should instead be "node only" .

As written, it looks like there is no fault tolerance in single region.